gwt-ol3-client/src/main/java/ol/proj/Projection.java

```diff
@@ -236,4 +237,21 @@ public Projection(ProjectionOptions projectionOptions) {}
 	@JsMethod(name = "transformExtent", namespace = "ol.proj")
     public static native Extent transformExtent(Extent extent, String source, String destination);
 
+
+	/**
+	 * Creates a new {@link Projection} for the given SRS-code. The projection
+	 * should have been registered with
+	 * {@link OLFactory#registerSpatialReference(String, String)} first!
+	 *
+	 * @param code combination of authority and identifier such as "EPSG:4326"
+	 * @return an {@link Projection} for the given code
+	 * @see {@link OLFactory#registerSpatialReference(String, String)}
+	 */
+	@JsOverlay
+	public static Projection getInstance(String code) {
```

Comment:
@peterza  Why this method is needed? There is already the method ´get()´ which do the job. Are there any differences?